OCAT/client/resources/js/components/shared/services/assessment.service.js

```diff
@@ -3,13 +3,12 @@ import axios from "axios";
 export class AssessmentService {
     static async submit (assessment) {
         try {
-            //Choose the correct method, url, and data to send in a request to the express OCAT/server/routes
-            await axios({});
+            await axios.post('http://localhost:3000/assessment/submit', assessment);
```

Comment:
```suggestion
            await axios.post('config.api.url/assessment/submit', assessment);
```
You can use your config.json for this so you don't share your host